src/address_form_config/address_book.js

```diff
@@ -29,6 +30,18 @@ export default [
         postcode: "#FormField_13"
       },
       stateMappings: defaultStateMappings
+    },
+    int: {
+      countryValue: {'Belgium': 'be', 'Canada': 'ca', 'Czechia': 'cz', 'Czech Republic': 'cz', 'Denmark': 'dk', 'France': 'fr', 'Germany': 'de', 'Ireland': 'ie', 'Netherlands': 'nl', 'Portugal': 'pt', 'Singapore': 'sg', 'Spain': 'es', 'Sweden': 'se', 'United Kingdom': 'gb', 'United States': 'us'},
+      elements: {
+        address1: '#FormField_8',
+        address2: '#FormField_9',
+        suburb: '#FormField_10',
+        state: '#FormField_12',
+        postcode: '#FormField_13',
+      },
+      stateMappings: internationalStateMappingsToNames,
+      optionalElements: {'be': ['address_line_2', 'state_territory'], 'ca': ['address_line_2'], 'cz': ['address_line_2', 'state_territory'], 'de': ['address_line_2'], 'fr': ['address_line_2', 'state_territory'], 'dk': ['address_line_2', 'state_territory'], 'ie': ['address_line_2'], 'nl': ['address_line_2', 'state_territory'], 'pt': ['address_line_2', 'state_territory'], 'sg': ['address_line_2', 'state_territory'], 'es': ['address_line_2'], 'se': ['address_line_2', 'state_territory'], 'gb': ['address_line_2', 'state_territory'], 'us': ['address_line_2']}
```

Comment:
Optional elements mean that the widget will not constantly reload when these elements are missing for these countries.